src/book.java

```diff
@@ -4,62 +4,62 @@
 @SuppressWarnings("serial")
 public class book implements Serializable {
 	
-	private String TITLE;
-	private String AUTHOR;
-	private String CALLNO;
-	private int ID;
+	private String title; // was TITLE
+	private String author; // was AUTHOR
+	private String callNo; // was CALLNO
+	private int id; // was ID
 	
-	private enum STATE { AVAILABLE, ON_LOAN, DAMAGED, RESERVED };
-	private STATE State;
+	private enum State {AVAILABLE, ON_LOAN, DAMAGED, RESERVED}; //delete whitespace, STATE
+	private State State;
```

Comment:
"private State State" should be changed to "private State state".

Reply:
great! has been now updated